### PR TITLE
changes test from 17.6

### PR DESCRIPTION
**Pull request by chenFriedman**

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts [hunk 1]
```diff
@@ -61,37 +61,70 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
...
-                IDnum: '.T0901828',
-                DOB: '24011971',
-                Tel: '0542987778'
-            },
-            {
-                IdType: 2,
-                IDnum: '?0901788',
-                DOB: '24011971',
-                Tel: '0542987778'
+        // const MOHHeader= {
```
Comment (Noam6421) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts:64]: remove lines 64-96

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts [hunk 2]
```diff
@@ -110,6 +143,7 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
         .finally(() => setIsLoading(false));
     };
 
+
```
Comment (Noam6421) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts:146]: remove

File: server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts [hunk 3]
```diff
@@ -11,22 +11,52 @@ const errorStatusCode = 500;
...
+        .then((result) => {
+            rulerLogger.info(launchingAPIRequestLog(result), Severity.LOW);
+            // rulerLogger.info(validAPIResponseLog, Severity.LOW);
+            // setToCache(req.originalUrl, result.result);
+            res.send(result);
         }
         ).catch((err: any) => {
             rulerLogger.error(invalidAPIResponseLog(err), Severity.HIGH);
-            res.status(errorStatusCode).send(err);
+            res.send(err);
```
Comment (Noam6421) [inline server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts:59]: return the errorstatuscode

Comment (chenFriedman) [reply server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts:59]: it returns error

**Review by barnahum8 — APPROVED** [review]

**Pull request merged**